mne_bids/mne_bids.py

```diff
@@ -689,18 +702,19 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
                overwrite, verbose)
 
     # TODO: Implement coordystem.json and electrodes.tsv for EEG and  iEEG
-    if kind == 'meg':
+    if kind == 'meg' and not emptyroom:
         _coordsystem_json(raw, unit, orient, manufacturer, coordsystem_fname,
                           overwrite, verbose)
 
     events = _read_events(events_data, raw)
-    if len(events) > 0:
+    if len(events) > 0 and not emptyroom:
```

Comment:
actually it would be nice to allow `events=None` ?

Reply:
doesn't `_read_events` just return an empty list if there are none?
`events_data` can be None however.

Reply:
sorry I meant `events_data`. If it can be `None`, then why do you need the `and not emptyroom` ?

Reply:
okay I guess this is also okay.